### PR TITLE
fixed broken image paths

### PR DESCRIPTION
**Pull request by braymundo**

### CODE REVIEW

File: support/README.md [hunk 1]
```diff
@@ -16,16 +16,16 @@ We are currently using OTRS as our main support system. Customers can either cre
 The basic support workflow is defined in the chart below. First level support is done mainly by our partners at [WOW24-7](https://wow24-7.io/). Issues are escalated to higher levels as appropriate:
 <br>
 <br>
-![](/images/support/support_flow.svg)
+![](https://rocket.chat/handbook/images/support/support_flow.svg)
 <br>
 Closed tickets also have their own workflow in order to ensure actionable items for each type of issue raised by customers:
 
-![](/images/support/support_closed.svg)
+![](ttps://rocket.chat/handbook/images/support/support_closed.svg)
```
Comment (marceloschmidt) [inline support/README.md]: `https` instead of `ttps`

File: support/README.md [hunk 2]
```diff
@@ -16,16 +16,16 @@ We are currently using OTRS as our main support system. Customers can either cre
...
 
 In summary closed tickets should have one of three possible outcomes:
 
 - If it's a **usability** issue, an existing document should be linked to the ticket and/or updated to reflect new knowledge
 - If it's a **functionality** issue, an existing GitHub feature request should be linked to the ticket and updated or a new one created to raise visibility to the customer request
 - If it's a **reliability** issue, an existing GitHub bug should be linked to the ticket and updated or a new one created to raise visibility to the problem reported
 
-![](/images/support/support_paths.svg)
+![](ttps://rocket.chat/handbook/images/support/support_paths.svg)
```
Comment (marceloschmidt) [inline support/README.md]: `https` instead of `ttps`

**Comment by braymundo** [comment]
Good question. Relative path worked fine on my local instance but failed when I pushed to remote... but I'm probably doing something wrong.

**Comment by marceloschmidt** [comment]
Did you try relative paths using `./images` or only absolute paths `/images`?

**Pull request closed without merging**